stackdriver-errors.js

```diff
@@ -46,6 +46,10 @@
     if(!config.projectId) {
       throw new Error('Cannot initialize: No project ID provided.');
     }
+    if(typeof StackTrace === 'undefined') {
+      // Inform about missing dependency
+      throw new Error('Cannot initialize: StackTrace object is missing, check if concat.dist.js version of stackdriver-errors was loaded');
```

Comment:
can you use "make sure you loaded "dist/stackdriver-errors-concat.js" or "dist/stackdriver-errors-concat.min.js", or that you imported the "stacktrace-js" module"